doc/source/developing/extensions.rst

```diff
@@ -35,6 +35,24 @@ In subsequent versions, we plan to include in yt a catalog of known extensions
 and where to find them; this will put discoverability directly into the code
 base.
 
+Frontend as an extension
+------------------------
+
+Starting with version 4.2 of yt, any externally installed package that exports
+:class:`~yt.data_objects.static_output.Dataset` subclass as an entrypoint in
+``yt.frontends`` namespace:
+
+.. code-block:: python
+
+   setup(
+       entry_points={
+           "yt.frontends": ["myFrontend = my_frontend.api.MyFrontendDataset"]
+       }
+   )
```

Comment:
it should be made explicit that this is an example using `setup.py`. I suggest also showing the syntax for `pyproject.toml` (example: https://github.com/neutrinoceros/yt_idefix).

`setup.cfg` is now legacy so it's not worth adding.

Reply:
Also, I assume from the type (list) that it's possible to define more than one Dataset subclass, but can you confirm ?

Reply:
That I don't know, but I believe so.

Reply:
It works with multiple entries.